src/can_service.py

```diff
@@ -141,8 +141,11 @@ def read_can(execution_flag, config_flag, init_flags, can_lock):
     customLogger.debug("CAN process started!")
 
     period = 2
-    period_counter = 0
-    previous_message_counter = 0
+    period_counter = 0  # if the counter reaches 5, 10 seconds have passed, and then the check is made whether the bus
```

Comment:
Looks good, just move the comments above the variable. Same goes for other variables.